apis/core/v1alpha1/resource_metadata.go

```diff
@@ -30,4 +30,7 @@ type ResourceMetadata struct {
 	// OwnerAccountID is the AWS Account ID of the account that owns the
 	// backend AWS service API resource.
 	OwnerAccountID *AWSAccountID `json:"ownerAccountID"`
+	// AWSRegion is the AWS region in which the resource is expected to be
+	// managed.
```

Comment:
instead of "which the resource is expected to be managed", this should read "which the resource exists". `Status.ACKResourceMetadata.Region` should only be set when a successful `Create` operation has occurred.